pensjon-brevbaker/src/main/kotlin/no/nav/pensjon/etterlatte/maler/barnepensjon/innvilgelse/BarnepensjonInnvilgelse.kt

```diff
@@ -41,13 +41,14 @@ import no.nav.pensjon.etterlatte.maler.vedlegg.barnepensjon.informasjonTilDegSom
 data class BarnepensjonInnvilgelseDTO(
     override val innhold: List<Element>,
     val beregning: BarnepensjonBeregning,
-    val etterbetaling: BarnepensjonEtterbetaling?,
-    val brukerUnder18Aar: Boolean,
     val bosattUtland: Boolean,
-    val kunNyttRegelverk: Boolean,
+    val brukerUnder18Aar: Boolean,
```

Comment:
Håper vi bruker named arguments heile konsekvent på Gjenny-sida for desse DTO-ane 😅

Reply:
Brukes konsekvent for innvilgelse og revurdering, så det skal være good 😄